tests/console/perl_bootloader.pm

```diff
@@ -49,12 +53,12 @@ sub run {
     assert_script_run 'pbl --del-option "TEST_OPTION"';
     assert_script_run('! grep -q "TEST_OPTION" /etc/default/grub');
 
-    # Create new log file and check if it exists and not empty
-    assert_script_run 'pbl --log /var/log/pbl-test.log';
-    if (is_tumbleweed && script_run('grep pbl /var/log/pbl-test.log') != 0) {
-        record_soft_failure('bsc#1219347 - pbl log creates empty log file');
-    } else {
-        validate_script_output 'cat /var/log/pbl-test.log', qr/pbl/;
+    # Add new option and check if it's logged in new log file
+    assert_script_run 'pbl --log /var/log/pbl-test.log --add-option LOG_OPTION="log_value"';
```

Comment:
This does not work on `$new_pbl` ?

Reply:
`pbl --log` should work everywhere.

I am using `$new_pbl` to test --default-settings which is a new feature.

Reply:
ok